package.json

```diff
@@ -9,7 +9,8 @@
   "scripts": {
     "watch": "webpack --config ./webpack.config.js --watch",
     "build": "webpack --config ./webpack.config.js",
-    "test": "karma start"
+    "test": "karma start",
+    "lint": "eslint normandy client webpack.config.js karam.conf.js"
```

Comment:
that being said i'm sure there is a better way than `|| true`. that was my terrible 5 second fix.